unified-runtime/source/adapters/level_zero/common.hpp

```diff
@@ -256,7 +257,7 @@ struct ReferenceCounter {
 };
 
 // Base class to store common data
-struct _ur_object {
+struct _ur_object : ur_handle_base_t_<ur::level_zero::ddi_getter> {
```

Comment:
Likewise, `ur::level_zero::handle`. `_ur_object` is also a reserved name so even more reason to make this change.

Reply:
`_ur_object` wasn't added by this change, that's an existing level zero type.

Reply:
Is that a reason not to change it?